research-software/onetep/onetep.rst

```diff
@@ -10,43 +10,77 @@ ONETEP
 ONETEP (Order-N Electronic Total Energy Package) is a linear-scaling code for
 quantum-mechanical calculations based on density-functional theory.
 
-
 Useful Links
 ------------
 
 * ONETEP home page      https://www.onetep.org
 * ONETEP tutorials      https://www.onetep.org/Main/Tutorials
 * ONETEP documentation  https://www.onetep.org/Main/Documentation
 
-
 Using ONETEP on ARCHER2
 -----------------------
 
-ONETEP is licensed software. Please see the ONETEP
-`license page <https://www.onetep.org/Main/License>`__ for details.
-Users who wish to access the ONETEP package should submit a request via SAFE.
+**ONETEP is only available to users who have a valid ONETEP licence.**
+
+If you have a ONETEP licence and wish to have access to ONETEP on ARCHER2,
+please make a request via the SAFE, see:
+
+  - `How to request access to package groups <https://epcced.github.io/safe-docs/safe-for-users/#how-to-request-access-to-a-package-group>`__
 
+Please have your license details to hand.
 
 Running parallel ONETEP jobs
 ----------------------------
 
+The following script will run a ONETEP job using 2 nodes (256 cores). it
+assumes that the input file is called ``text_calc.dat``.
 
-.. warning::
+::
+
+  #!/bin/bash
+
+  # Request 2 nodes with 128 MPI tasks per node for 20 minutes
+  # Replace [budget code] below with your account code,
+  # e.g. '--account=t01'
 
-  ONETEP example scripts are pending
+  #SBATCH --job-name=ONETEP
+  #SBATCH --nodes=2
+  #SBATCH --tasks-per-node=128
+  #SBATCH --cpus-per-task=1
+  #SBATCH --time=00:20:00
 
+  #SBATCH --partition=standard
+  #SBATCH --qos=standard
+  
+  #SBATCH --account=[budget code]
+  #SBATCH --partition=standard
+  #SBATCH --qos=standard
 
-Hints and tips
+  # Load the ONETEP module
+  module load onetep
+
+  # Make sure that the stack settings are correct
+  ulimit -s unlimited
+  export OMP_STACKSIZE=64M
+  export OMP_NUM_THREADS=1
+
+  # Launch the executable
+  srun -cpu-bind=cores onetep.archer2 test_calc > test_calc.out
```

Comment:
It should contain `--cpu-bind=cores` (please note that a hyphen is missing in the original script).